src/turnCalendar.js

```diff
@@ -290,7 +290,10 @@ angular
         var pickValue = function (property) {
 
             if (angular.isDefined($attrs[property])) {
-                return $scope.$parent.$eval($attrs[property]) || $attrs[property];
+                if ($scope.$parent.$eval($attrs[property]) != null)
+                    return $scope.$parent.$eval($attrs[property]);
+                else
+                    return $attrs[property];
```

Comment:
If `$scope.$parent.$eval($attrs[property])` is an idempotent operation, it should be stored in a variable rather than computing it several times.

e.g.

```javascript
var value = $scope.$parent.$eval($attrs[property];
return value != null ? value : $attrs[property];
```